src/ioctls/vm.rs

```diff
@@ -1216,6 +1217,40 @@ impl VmFd {
     pub fn run_size(&self) -> usize {
         self.run_size
     }
+
+    /// Wrapper over `KVM_CHECK_EXTENSION`.
+    ///
+    /// Returns 0 if the capability is not available and a positive integer otherwise.
+    fn check_extension_int(&self, c: Cap) -> i32 {
```

Comment:
Are there no cases where you would need to get the result of calling `KVM_CHECK_EXTENSION` rather than just knowing it is available or not?

One example for KVM capabilities where you need to know the actual value returned by KVM_CHECK_EXTENSION is getting the kvm api number. The return of that ioctl should be 12 and you actually need to check that is 12, not that it is higher than 0.

My question comes from trying to understand if we need to make `check_extention_int` public instead of solely relying on `check_extension`.

Reply:
Looking into kernel code handling `KVM_CHECK_EXTENSION` (ARM only): https://github.com/torvalds/linux/blob/v5.4/virt/kvm/kvm_main.c#L3198 and https://github.com/torvalds/linux/blob/v5.4/virt/kvm/arm/arm.c#L180, I didn't find any numerical extension that have to be checked per VM. 

So I think we don't need to expose it public, now.